state-chain/runtime/src/chainflip.rs

```diff
@@ -253,6 +261,65 @@ impl TransactionBuilder<Bitcoin, BitcoinApi<BtcEnvironment>> for BtcTransactionB
 	}
 }
 
+pub struct ArbTransactionBuilder;
+
+impl TransactionBuilder<Arbitrum, ArbitrumApi<EvmEnvironment>> for ArbTransactionBuilder {
```

Comment:
wrote a macro to dedup the implementation with an additional trait to factor out the difference in impl. Going the other way by having one impl of TransactionBuilder for evm chains is more work and conceptually not great since Transaction builder is chain-specific and not chain crypto specific.